flags.ts

```diff
@@ -51,3 +51,16 @@ export const githubIntegrationFlag = flag<boolean>({
 		{ value: true, label: "Enable" },
 	],
 });
+
+export const freePlanFlag = flag<boolean>({
+	key: "free-plan",
+	async decide() {
+		return takeLocalEnv("FREE_PLAN_FLAG");
```

Comment:
Please add this flag to `.env.example` for easier development setup.

https://github.com/giselles-ai/giselle/blob/0f1f235457fb18e2125e611406fb081ec0fb4d14/.env.example#L4-L7